app/tests/integration/nimbus/test_remote_settings.py

```diff
@@ -1,9 +1,38 @@
+from urllib.parse import urlparse
+
 import pytest
+import requests
+from nimbus.pages.browser import Browser
 from nimbus.pages.experimenter.summary import SummaryPage
 from nimbus.pages.remote_settings.dashboard import Dashboard
 from nimbus.pages.remote_settings.login import Login
 
 
+def load_data():
+    apps = []
+    data = requests.post(
+        "https://nginx/api/v5/graphql",
+        json={
+            "operationName": "getConfig",
+            "variables": {},
+            "query": "\nquery getConfig {\n  nimbusConfig "
+            "{\n    targetingConfigs {\n      "
+            "label\n      value\n      applicationValues\n    "
+            "}\n  }\n}\n",
+        },
+        verify=False,
+    ).json()
+    for item in data["data"]["nimbusConfig"]["targetingConfigs"]:
+        if "DESKTOP" in item["applicationValues"]:
+            apps.append(item["value"])
+    return apps
```

Comment:
List comprehensions are your friend 👬 

```py
return [item for item in data["data"]["nimbusConfig"]["targetingConfigs"] if "DESKTOP" in item["applicationValues"]]
```

Reply:
Yeah this is something I am not good at lol

Reply:
Not sure if you want to do this or not. I'm indifferent to syntax if you think one is better or more legible than the other option.

Reply:
I like list comprehensions but sometimes I don't. Not sure here, I'll just add it.

Reply:
On second thought, the way I have the test setup is looking for the data structure I had. I don't think this will add much adding it.